contrib/wordlyworks/ner.py

```diff
@@ -0,0 +1,24 @@
+import spacy
+
+
+def ner():
+    nlp = spacy.load("en_core_web_sm")
+
+    text_description = '''
+    Apple Inc. is planning to open a new factory in Austin, Texas. 
+    The plan will reportedly cost $1 billion and create jobs for 5,000 people. 
+    Apple's CEO, Tim Cook, will oversee the new project.
+    '''
+
+    document_object = nlp(text_description)
+
+    for entity in document_object.ents:
+        print(f"{entity.text}, ({entity.label_})")
```

Comment:
No need to print, just return the relevant entities.

Also, please move this function to `main.py` and name it `get_entities`. It should also accept a string as an argument.